Sources/BraveWallet/Crypto/Accounts/Activity/AccountActivityView.swift

```diff
@@ -11,229 +11,359 @@ import DesignSystem
+          .accessibilityHidden(true)
+        VStack(spacing: 0) {
+          Text(store.account.name)
+            .font(.title2.weight(.semibold))
+          AddressView(address: store.account.address) {
+            Text(store.account.address.truncatedAddress)
+              .font(.caption)
+          }
         }
       }
-      .padding(.bottom, 12)
-      HStack {
-        Button(action: { action(false) }) {
-          HStack {
-            Image(braveSystemName: "leo.qr.code")
-              .font(.body)
-            Text(Strings.Wallet.detailsButtonTitle)
-              .font(.footnote.weight(.bold))
-          }
+      
+      Spacer().frame(height: 16)
+      
+      VStack {
+        if store.isLoadingAccountFiat {
+          Text(store.accountTotalFiat)
+            .font(.title.weight(.medium))
+            .redacted(reason: .placeholder)
+            .shimmer(store.isLoadingAccountFiat)
+        } else {
+          Text(store.accountTotalFiat)
+            .font(.title.weight(.medium))
         }
-        Button(action: { action(true) }) {
-          HStack {
-            Image(braveSystemName: "leo.edit.pencil")
-              .font(.body)
-            Text(Strings.Wallet.renameButtonTitle)
-              .font(.footnote.weight(.bold))
-          }
+        Text(store.account.accountSupportDisplayString)
+          .font(.caption)
+      }
+      
+      Spacer().frame(height: 24)
+      
+      HStack(spacing: 24) {
+        PortfolioHeaderButton(style: .buy) {
+          buySendSwapDestination = .init(kind: .buy)
         }
+        PortfolioHeaderButton(style: .send) {
+          buySendSwapDestination = .init(kind: .send)
+        }
+        PortfolioHeaderButton(style: .swap) {
+          buySendSwapDestination = .init(kind: .swap)
+        }
+      }
+    }
+    .padding(.vertical, 24)
+    .frame(maxWidth: .infinity)
+    .background(
+      Color(braveSystemName: .containerBackground)
+    )
+  }
+  
+  private var rowsSection: some View {
+    VStack(spacing: 0) {
+      NavigationLink(destination: {
+        AssetsListDetailView(
+          store: store,
+          cryptoStore: cryptoStore,
+          keyringStore: keyringStore
+        )
+      }, label: {
+        RowView(
+          iconBraveSystemName: "leo.crypto.wallets",
+          title: Strings.Wallet.assetsTitle,
+          description: String(
+            format: "%d %@",
+            store.userAssets.count,
+            Strings.Wallet.assetsTitle.lowercased()
+          )
+        )
+      })
+      Divider()
+      NavigationLink(destination: {
+        NFTGridDetailView(
+          store: store,
+          cryptoStore: cryptoStore,
+          keyringStore: keyringStore
+        )
+      }, label: {
+        RowView(
+          iconBraveSystemName: "leo.grid04",
+          title: Strings.Wallet.nftsTitle,
+          description: String(
+            format: "%d %@",
+            store.userNFTs.count,
+            Strings.Wallet.nftsTitle
```

Comment:
can we handle the singular/plural? Currently its 
1 Assets
1 NFTs
1 Transactions

Reply:
Updated copy in [001dd03](https://github.com/brave/brave-ios/pull/8716/commits/001dd03228118991496ab9a943241e1cd114042c)